plenum/server/node.py

```diff
@@ -1835,15 +1929,19 @@ def reportSuspiciousNode(self,
         #                        reason=InvalidSignature.reason,
         #                        code=InvalidSignature.code)
 
-        if code in self.suspicions:
-            # TODO: Reconsider tolerating some suspicions, and if you tolerate,
-            #  why are they suspicions?
-            if code not in (Suspicions.DUPLICATE_PPR_SENT,
-                            Suspicions.DUPLICATE_PR_SENT,
-                            Suspicions.DUPLICATE_CM_SENT):
-                self.blacklistNode(nodeName,
-                                   reason=self.suspicions[code],
-                                   code=code)
+        # TODO: Consider blacklisting nodes again.
```

Comment:
We blacklist on strictly malicious cases, since duplicate messages, certain messages from Primary or non-primary can also happen if network is unstable or a node is behind